app/views/report_mailer/registrar_import_email.html.erb

```diff
@@ -31,7 +31,7 @@
 
   <ul>
     <% @results[:new_degrees].each do |degree| %>
-      <li><%= degree %></li>
+      <li><%= link_to degree.name_dw, "/admin/degrees/#{degree.id}" %></li>
```

Comment:
I think these links will be problematic in an actual sent email (haven't confirmed yet).

We may need to look into this:
https://edgeguides.rubyonrails.org/action_mailer_basics.html#generating-urls-in-action-mailer-views